nomos-services/cryptarchia-consensus/src/storage/adapters/storage.rs

```diff
@@ -0,0 +1,87 @@
+// STD
```

Comment:
I think we don't use caps for this
```suggestion
// std
```

Reply:
For some reason it feels very weird to me to have it written in lowercaps, but I'll try not to look at it 😂